test/integration/metrics-manifests/rendered-test.yaml

```diff
@@ -1,25 +0,0 @@
-apiVersion: batch/v1
```

Comment:
this file should not be here. added to a `.gitignore` file for another PR for issue #143 which captured a few developer experience improvements (IMHO)